lib/capybara/node/actions.rb

```diff
@@ -258,13 +260,16 @@ def unselect(value = nil, from: nil, **options)
       #   @param [String] locator                  Which field to attach the file to
       #   @param [String, Array<String>] paths     The path(s) of the file(s) that will be attached
       #
-      #   @option options [Symbol] match (Capybara.match)     The matching strategy to use (:one, :first, :prefer_exact, :smart).
-      #   @option options [Boolean] exact (Capybara.exact)    Match the exact label name/contents or accept a partial match.
+      #   @option options [Symbol] match
+      #     The matching strategy to use. Defaults to {Capybara.configure match}.
+      #   @option options [Boolean] exact
+      #     Match the exact label name/contents or accept a partial match. Defaults to {Capybara.configure exact}.
       #   @option options [Boolean] multiple Match field which allows multiple file selection
       #   @option options [String, Regexp] id             Match fields that match the id attribute
       #   @option options [String] name           Match fields that match the name attribute
       #   @option options [String, Array<String>, Regexp] class    Match fields that match the class(es) provided
-      #   @option options [true, Hash] make_visible   A Hash of CSS styles to change before attempting to attach the file, if `true` { opacity: 1, display: 'block', visibility: 'visible' } is used (may not be supported by all drivers)
+      #   @option options [true, Hash] make_visible
+      #     A Hash of CSS styles to change before attempting to attach the file, if `true`, `{ opacity: 1, display: 'block', visibility: 'visible' }` is used (may not be supported by all drivers).
```

Comment:
[note] I apply the Markdown style code syntax (using backslash) as possible. Because the Markdown style will be adopted in the future and it's more familiar than the YARD style for most developers and contributors.

See also:
https://github.com/teamcapybara/capybara/blob/0f5fb07d9b7954ae3bd0d0924dcfb150d8e8ee4c/Rakefile#L44

For example:

```diff
- if +true+,
+ if `true`,
```

For details, see <https://github.com/teamcapybara/capybara/pull/2198#discussion_r286291677>.